src/node_binding.cc

```diff
@@ -87,6 +86,7 @@
   V(zlib)
 
 #define NODE_BUILTIN_MODULES(V)                                                \
+  NODE_BUILTIN_ISOLATE_AWARE_MODULES(V)                                        \
```

Comment:
It seems a bit weird to reuse the list here for the per-isolate registration functions, considering the ids are in conflict with the actual module registration functions..can we create another map or linked list to store this?

Reply:
It's a list of internal modules that are supposed to to have `_register##modname()` called in `binding::RegisterBuiltinModules()`. But I think we should have a separate list for these, because they don't necessarily overlap (e.g. a module can register templates without having a binding of its own, e.g. 'process_object')